frontend/src/dialog/photo/edit/files.vue

```diff
@@ -308,6 +313,12 @@ export default {
     openFile(file) {
       this.$viewer.show([Thumb.fromFile(this.model, file)], 0);
     },
+    getFolderUri(file) {
+      const fileName = file.Name;
+      const folder = fileName.substring(0, fileName.lastIndexOf('/'));
+
+      return(this.config.baseUri + '/library/index/files/' + folder);
```

Comment:
FYI, you can use template literals with \` like this:
``return `${this.config.baseUri}/library/index/files/${file.name.substring(0, file.name.lastIndexOf('/'))}`;``
See [the mozilla doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Reply:
I think that it's easier to read with 2 variables.